modules/VolcanoPlot.nf

```diff
@@ -0,0 +1,34 @@
+nextflow.enable.dsl=2
+
+def s = """\
+        _________________________________________________
+        Nextflow Volcano:
+
+        ▶ Input DESeq2 IHW results: ${params._DESeq2_result.tsv}
```

Comment:
This won't work, as the deseq2 res is not a param (which are provided by the user), but a channel which comes from the previous process. 

In fact, you could remove this entire logging message. It makes more sense to handle logging by the main script, and not by  the modules (this also applies to the DESeq2 module)